core/validation.go

```diff
@@ -162,7 +162,7 @@ func (app *App) validateOrders(orders []*zeroex.SignedOrder) (*ordervalidator.Va
 
 		// Check if order is already stored in DB
 		var dbOrder meshdb.Order
-		err = app.db.Orders.FindByID(orderHash.Bytes(), &dbOrder)
+		err = app.orderSelector.db.Orders.FindByID(orderHash.Bytes(), &dbOrder)
```

Comment:
This could also be changed if we re-add the `db` field to `app`.

```suggestion
		err = app.db.Orders.FindByID(orderHash.Bytes(), &dbOrder)
```